sql/expression/function/aggregation/count.go

```diff
@@ -82,14 +79,16 @@ func (c *Count) Update(ctx *sql.Context, buffer, row sql.Row) error {
 
 // Merge implements the Aggregation interface.
 func (c *Count) Merge(ctx *sql.Context, buffer, partial sql.Row) error {
-	span, _ := ctx.Span("aggregation.Count_Merge")
-	defer span.Finish()
-
 	buffer[0] = buffer[0].(int32) + partial[0].(int32)
 	return nil
 }
 
 // Eval implements the Aggregation interface.
 func (c *Count) Eval(ctx *sql.Context, buffer sql.Row) (interface{}, error) {
-	return buffer[0], nil
+	span, ctx := ctx.Span("aggregation.Count_Eval")
+	count := buffer[0]
+	span.LogKV("count", count)
```

Comment:
why do we need the result of the aggregation in the log?

Reply:
...just to be consistent with other operations (max, min, avg, ...) if it's too noisy we can remove it.

Reply:
Yeah, perhaps having in the logs the results of these operations is too noisy